src/rules/border-styles.ts

```diff
@@ -0,0 +1,9 @@
+import { RuleFn } from "./RuleConfig";
+import { makeRules } from "../utils";
+
+// http://tachyons.io/docs/themes/borders/
+export const borderStyleRules: RuleFn = () =>
+  makeRules("borderStyle", {
+    b__dashed: "dashed",
+    b__dotted: "dotted",
```

Comment:
Add `solid` and `none`?

Reply:
I did a quick check and those already exist (`ba` and `bn`).

Reply:
Hrm, it looks like `bn` both both borderStyle + borderWidth 0...i.e. it's "border none", so it's probably fine to also add this has "very specifically just change order style".

Reply:
Right - it's a little tricky since the `ba` and `bn` rules are compound rules. I am not sure of the use cases for `bt`, `br`, `bb`, `bl` too.

The namespace of `b` is open (I thought it was previously taken by bottom but that is using `bottom`). What if we remove the `ba` with `b` to represent the basic border?

Reply:
I am thinking of adding the `bw<increment>` rule too.

Reply:
I think `ba` == "border all". Kinda ambivalent/would probably leave as is.

Add bwIncrement seems fine, although I wonder if that should be `bw<Pixel>` because are people really going to want like 8x, 16px, 24px borders? :-) 

Another option is that we could lean on TypeScript and make these more method-ish, i.e.:

```Css.ba({ width: 3 })``` would mean "border all but use a border-width of 3px".

It'd take a little more work on the Truss side to define that rule because it's not a strait "value -> value" mapping like most of them are...